include/openthread/instance.h

```diff
@@ -53,7 +53,7 @@ extern "C" {
  * @note This number versions both OpenThread platform and user APIs.
  *
  */
-#define OPENTHREAD_API_VERSION (1)
+#define OPENTHREAD_API_VERSION (2)
```

Comment:
Why does this PR or #4935 require a version bump? APIs have historically been constrained to [`include/`](https://github.com/openthread/openthread/tree/master/include). We need to have a clear description of the API surface and all such APIs documented in https://openthread.io/reference

Reply:
Actually it's not for this PR, but the #4935 added posix platform API otSysGetRadioUrlHelpString(). I'm not sure if this should cause increment of this version number.

Reply:
I reverted increasing the version number.